packages/opentelemetry-sampler-aws-xray/src/remote-sampler.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Sampler,
+  SamplingDecision,
+  SamplingResult,
+} from '@opentelemetry/sdk-trace-base';
+import { diag } from '@opentelemetry/api';
+import { SamplingRule, AWSXRaySamplerConfig } from './remote-sampler.types';
+import axios from 'axios';
+
+const DEFAULT_INTERVAL_MS = 5 * 60 * 1000; // 5 minutes on sampling rules fetch (default polling interval)
+const DEFAULT_ENDPOINT = 'http://localhost:2000';
```

Comment:
This is the default endpoint that the `awsproxy` in the OTel collector is listening on, which is why it is set as the default in this implementation as well. It can also be configured to be any other endpoint depending on the use case/application 
https://aws-otel.github.io/docs/getting-started/remote-sampling#enable-awsproxy-extension